CHANGELOG.md

```diff
@@ -4950,6 +4950,10 @@
   - ### Features
     - Amazon Transcribe - support transcriptions from audio sources in Russian (ru-RU) and Chinese (zh-CN).
 
+## __AWS SDK for Java__
```

Comment:
You can't edit this file directly because it will be overwritten when the new is generated.